### ISSUE TITLE
Parameter/Variable Is Not Used Misses One

### ISSUE THREAD

**Issue by iakoder** [kalessil/phpinspectionsea#1004]
First, thank you for reporting a bug or making a request. That takes time and we appreciate that!

Let's start with common details:
 
| Subject    | Details                                                  |
| :--------- | :------------------------------------------------------- |
| Issue type | Bug/False-Positive    |
| IDE        | PhpStorm 2018.2.2 EAP                          |
| Plugin     | Php Inspections (EA Extended), 3.0.5.1 |

```
<?php

function f($p) {

    $a = $b = [];

    foreach ($p as $aP) {

        if ($aP < 0) {
            // $a is not flaged as Parameter/variable is overridden but is never used or appears outside of the scope
            $a[] = $aP;
        } else {
            // $b is flaged as Parameter/variable is overridden but is never used or appears outside of the scope
            $b[] = $aP;
        }

    }

}
```

PhpStorm 2018.2.2
Build #PS-182.4129.45, built on August 22, 2018
Licensed to Intacct / Dan Wilks
Subscription is active until June 23, 2019
JRE: 1.8.0_152-release-1248-b8 x86_64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
macOS 10.13.6

**Comment by kalessil** [comment]
Confirmed, thank you @iakoder !

**Comment by kalessil** [comment]
Fixed!
